lib/yabeda/datadog/worker/send.rb

```diff
@@ -18,7 +18,9 @@ class Worker
             value = payload.fetch(:value)
             tags = payload.fetch(:tags)
 
-            stats.send(metric.type, metric.name, value, tags: tags)
+            Logger.instance.log_request metric do
+              stats.send(metric.type, metric.name, value, tags: tags)
+            end
```

Comment:
This will not work because of `stats.send(metric.type` end up to call [UDPSocket#send](https://devdocs.io/ruby~2.5/udpsocket#method-i-send) which returns a number of bytes sent. Maybe it not required to catch errors because `dogstatsd-ruby` already do [logging](https://github.com/DataDog/dogstatsd-ruby/blob/master/lib/datadog/statsd.rb#L43).

Reply:
Yes, i agree. I added logger there just to show that request was sent and returned something.